documents-storage-jcr/src/main/java/org/exoplatform/documents/storage/jcr/util/JCRDocumentsUtil.java

```diff
@@ -212,7 +212,7 @@ public static FolderNode toFolderNode(IdentityManager identityManager,
       }
       folderNode.setDatasource(JCR_DATASOURCE_NAME);
       folderNode.setPath(node.getPath());
-      folderNode.setCloudDriveFolder(node.hasProperty("ecd:connected"));
+      folderNode.setCloudDriveFolder(node.hasProperty("ecd:connected") || node.hasProperty("ecd:driveUUID"));
```

Comment:
``
      || node.hasProperty("ecd:driveUUID"));
``
this will affect also sub folders, we should verify that